airsonic-main/cve-suppressed.xml

```diff
@@ -9,6 +9,12 @@
     </suppress>
     <!-- << Provisional -->
 
+    <suppress>
+        <notes><![CDATA[CGI is not used]]></notes>
```

Comment:
What about `<notes>Tomcat isn't used in CGI mode, so we're not affected.</notes>` instead?

Reply:
We have approved your request.